src/app/components/wrapper-components/part-creation/part-creation.component.spec.ts

```diff
@@ -708,17 +780,34 @@ describe('PartCreationComponent', () => {
                 expect(component.stopSendingPresenceTokensAndObservingUsersIfNeeded).not.toHaveBeenCalled();
                 component.stopSendingPresenceTokensAndObservingUsersIfNeeded();
             }));
+            it('should update observedPart and mark user as chosen opponent', fakeAsync(async() => {
+                // Given a partCreation where user is candidate
+                awaitComponentInitialization();
+
+                // When user is selected as chosen opponent
+                spyOn(observedPartService, 'updateObservedPart').and.callThrough();
+                await receiveConfigRoomUpdate(ConfigRoomMocks.WITH_CHOSEN_OPPONENT);
+
+                // Then an update should change user doc to say it's chosenOpponent now
+                expect(observedPartService.updateObservedPart)
+                    .toHaveBeenCalledOnceWith(FocusedPartMocks.CHOSEN_OPPONENT);
+                // To avoid finishing test with periodic timer in queue
+                component.stopSendingPresenceTokensAndObservingUsersIfNeeded();
+            }));
+        });
+        describe('Someone else is chosen', () => {
+            it('should update');
         });
```

Comment:
test vide pour un futur ticket ?

Reply:
Ah oui, un bébé ticket "when candidate and someone else is chosen as opponent, change the FocusedPart role of the current user", tpr.